src/wasm2c.c.tmpl

```diff
@@ -63,12 +76,88 @@ DEFINE_STORE(i64_store8, u8, u64);
 DEFINE_STORE(i64_store16, u16, u64);
 DEFINE_STORE(i64_store32, u32, u64);
 
+#if _MSC_VER
+#if _M_IX86
+static inline unsigned long LowDword(unsigned __int64 value) {
+  return (unsigned long)value;
+}
+
+static inline unsigned long HighDword(unsigned __int64 value) {
+  unsigned long high;
+  memcpy(&high, (unsigned char*)&value + sizeof(high), sizeof(high));
+  return high;
```

Comment:
Why not `return value >> 32;`?

Reply:
True, I copied this from config.h.in, but I'm not sure why I wrote it this way. :-)